conf/messages.cy

```diff
@@ -71,9 +71,13 @@ error.submissionError.link = ddechrau’ch hawliad eto
 
 # Session Expired
 # ----------------------------------------------------------
-session_expired.title = Er eich diogelwch, mae’r gwasanaeth hwn wedi’i ailosod
-session_expired.heading = Er eich diogelwch, mae’r gwasanaeth hwn wedi’i ailosod
-session_expired.guidance = Mae’r manylion a roddwyd gennych wedi’u dileu gan nad ydych wedi mynd yn eich blaen â’r gwasanaeth ers 15 munud.
+pageNotFound.title = (W)Page not found - Error - Register your charity’s details with HMRC - GOV.UK(W)
```

Comment:
remove  **- Register your charity’s details with HMRC - GOV.UK(W)**.. this will create duplicate in title